children/schema.py

```diff
@@ -89,8 +92,11 @@ class Input:
     @login_required
     @transaction.atomic
     def mutate_and_get_payload(cls, root, info, **kwargs):
-        user = info.context.user
+        children_data = kwargs["children"]
+        if not children_data:
+            raise GraphQLError("At least one child is required.")
```

Comment:
isn't it ensure by the changes you made to the inout args?

Reply:
maybe [like this](https://docs.graphene-python.org/en/latest/types/list-and-nonnull/#nonnull-lists)?

Reply:
Yeah, AFAIK there is no way in GraphQL schema to force a list to have at least one item.

But your suggestion of forbidding nulls is also needed, we don't want to allow `[null]`, so added that :)

Reply:
hm, wouldn't the way you did it now enforce `a list to have at least one item`?
you get right now the following schema: `children: [ChildInput!]!`

Reply:
I think this bit is now redundant:
```
children_data = kwargs["children"]
if not children_data:
    raise GraphQLError("At least one child is required.")
```

Reply:
Yes I get exactly that schema now, but it still allows `children: []` so that check is needed.

Reply:
is this a bug in graphene? 😅 man, well, we can keep the check I guess.

Reply:
It is actually how it goes in GraphQL, see [this](https://stackoverflow.com/questions/46770501/graphql-non-nullable-array-list/46771890#46771890).

Reply:
oh, I see! thanks for the link. I had a different understanding of those `!` marks. 🙂 this PR is then 💯 ⚡️